scripts/check-sidebar-lazy-layout.py

```
@@ -1,741 +1,499 @@
-        except OSError:
-            continue
-        if "Layout" not in text:
-            continue
-        for match in CUSTOM_LAYOUT_DECL.finditer(neutralize_swift(text)):
-            names.add(match.group(1))
-    return names
+        next_visited = visited | {name}
+        for helper_body in function_bodies[name]:
+            path = mutation_path(helper_body, function_bodies, next_visited)
+            if path is not None:
+                return [name] + path
```

Ruff (0.15.21)

[warning] Consider `[name, *path]` instead of concatenation

Replace with `[name, *path]`

(RUF005)